pkg/crs/env.go

```diff
@@ -0,0 +1,33 @@
+package crs
+
+import "github.com/stackrox/rox/pkg/env"
+
+const (
+	// LegacySensorServiceCertEnvName contains the environment variable name used by the CRS-flow to check for the
+	// existence of a legacy sensor service certificate (e.g. coming from an init-bundle).
+	// Needs to be kept in sync with
+	// image/templates/helm/stackrox-secured-cluster/templates/sensor.yaml.htpl:spec.template.spec.initContainers[0].env.
```

Comment:
Are you adding a cross-reference there to here in the followup PR?

Reply:
yes.